app/views/activities/index.html.erb

```diff
@@ -11,6 +11,10 @@
   </nav>
 <% end %>
 
+<div class="content-container mt-3 py-3 activity-filters">
```

Comment:
```suggestion
<div class="content-container mt-3 py-3">
```

do we need this class? I don't see it being used.

Reply:
I used it while testing and forgot it there.I removed it now.